src/main/java/com/iexec/worker/docker/ComputeMeta.java

```diff
@@ -0,0 +1,22 @@
+package com.iexec.worker.docker;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/* /!\ this probably should be split */
```

Comment:
Yes, we probably need to merge/split/clean DockerExecutionResult, ResultInfo & ComputeMeta.

Reply:
At first, I wanted to use one big `Computation` structure to replace all of them but that would've complicated the refactoring and the PR. It will absolutely come in the next iterations.